test/controllers/color_controller_test.exs

```diff
@@ -0,0 +1,16 @@
+defmodule Elixirlight.ColorControllerTest do
+  use Elixirlight.ConnCase
+
+  defstruct [:color]
+
+  test "GET /api/healthcheck", %{conn: conn} do
+    conn = get conn, "/api/healthcheck"
+    assert json_response(conn, 200)
+  end
+
+  test "GET /api/color", %{conn: conn} do
+    color_as_json = Poison.encode!(%{color: "#ff0000"})
+    conn = get conn, "/api/color"
+    assert json_response(conn, 200) =~ color_as_json
```

Comment:
Also, I've usually seen:
```
response = get conn, "/api/color"
```
Just so it is easy to differentiate that it is now a response object that is already been through the connection.